packages/app/src/cli/models/app/loader.ts

```diff
@@ -389,6 +389,10 @@ class AppLoader<TConfig extends AppConfiguration, TModuleSpec extends ExtensionS
       usedCustomLayoutForExtensions: configuration.extension_directories !== undefined,
     })
 
+    if (process.env.REMOTE_DOM_EXPERIMENT) {
+      await appClass.generateExtensionTypes()
```

Comment:
can we measure how long this takes? for both cases where we don't have a type file and when we already have one.
(best to use `const start = startHRTime()` and `endHRTimeInMs(start)` for accurate values)